projects/bp-gallery/src/components/views/start/StartView.tsx

```diff
@@ -48,25 +48,34 @@ const StartView = () => {
     fallback: { clientId: '', donationText: '', purposeText: '' },
   });
 
-  const archiveCards = archives?.map(archive => {
-    const sharedProps = {
-      archiveName: archive.name,
-      archiveDescription: archive.shortDescription ?? '',
-      archiveId: archive.id,
-      archivePictureCount: pictureCounts?.find(pictureCount => pictureCount.id === archive.id)
-        ?.count,
-    };
+  const archiveCards = useMemo(
+    () =>
+      archives
+        ?.filter(archive => isLoggedIn || !archive.hidden)
+        ?.map(archive => {
+          const sharedProps = {
+            archiveName: archive.name,
+            archiveDescription: archive.shortDescription ?? '',
+            archiveId: archive.id,
+            archivePictureCount: pictureCounts?.find(pictureCount => pictureCount.id === archive.id)
+              ?.count,
+          };
 
-    return (
-      <div className='archive' key={archive.id}>
-        {archive.showcasePicture ? (
-          <ArchiveCard picture={archive.showcasePicture} {...sharedProps} />
-        ) : (
-          <ArchiveCardWithoutPicture {...sharedProps} />
-        )}
-      </div>
-    );
-  });
+          return (
+            <div
+              className={`archive ${archive.hidden ? 'brightness-90 opacity-50' : ''}`}
```

Comment:
Is archive necessary for styling?

Reply:
You are right, it's unnecessary. I removed it :) (62be208efc5ab1edae9935b16100da717fdf96c8)